README.md

```diff
@@ -204,45 +213,45 @@ configure script and makefile (the latter of which just invokes `x.py`).
 make && sudo make install
 ```
 
-When using the configure script, the generated `config.mk` file may override the
-`config.toml` file. To go back to the `config.toml` file, delete the generated
-`config.mk` file.
+When using the configure script, the generated `config.mk` file may override
+the `config.toml` file. To go back to the `config.toml` file, delete the
+generated `config.mk` file.
 
 ## Building Documentation
 
-If you’d like to build the documentation, it’s almost the same:
+If you'd like to build the documentation, it's almost the same:
 
 ```sh
 ./x.py doc
 ```
 
-The generated documentation will appear under `doc` in the `build` directory for
-the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory will be
-`build\x86_64-pc-windows-msvc\doc`.
+The generated documentation will appear under `doc` in the `build` directory
+for the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory
```

Comment:
Awkward Latin abbreviation in the middle of nowhere, worsens readability for no good reason, doesn't have to be changed her but if you have an opinion on what else could go here I'd be happy to see it.
```suggestion
for the ABI used. That is, if the ABI was `x86_64-pc-windows-msvc`, the directory
```